src/com/esotericsoftware/kryo/io/OutputChunked.java

```diff
@@ -58,8 +58,9 @@ public void flush () throws KryoException {
 			} catch (IOException ex) {
 				throw new KryoException(ex);
 			}
+		}else{

```

Comment:
Please apply project formatting to your changes. There should be whitespace around the `else`.

Reply:
thank you , i have formtted